src/perl/Permabit/VolumeGroup.pm

```diff
@@ -445,10 +445,41 @@ sub getFreeBytes {
 }
 
 ########################################################################
-# Return an LVM config argument to use in lvm command. Adds in an
-# additional config filter to strip out test devices such as
-# Permabit::BlockDevice::TestDevice::Fua in order to prevent LVM
-# commands from seeing duplicate PVs.
+# Return an LVM devices string to use in lvm command. Function will
+# filter out devices below test devices so as to avoid duplicate PVs.
+#
```

Comment:
This description seems to imply the function will filter out any device below a TestDevice, but it only does so if the TestDevice is immediately below the VolumeGroup in the stack. I presume this doesn't make a difference for the stacks we use in our tests, but I think the description should at least be accurate. 
(I also concede this matches what the filtering used to do.)

Reply:
I'll clean up the doc